GMX-Bots/Detect-profitable-trades/src/agent.spec.ts

```diff
@@ -0,0 +1,427 @@
+import { FindingType, FindingSeverity, Finding, ethers, HandleTransaction, TransactionEvent } from "forta-agent";
+import { TestTransactionEvent } from "forta-agent-tools/lib/tests";
+import { provideHandleTx } from "./agent";
+import { createAddress } from "forta-agent-tools/lib/tests";
+import { Interface } from "@ethersproject/abi";
+
+const ABI: string[] = [
+  "event Swap(address account, address tokenIn, address tokenOut, uint256 amountIn, uint256 amountOut)",
```

Comment:
```suggestion
  "event Swap(address account, address tokenIn, address tokenOut, uint256 amountIn, uint256 amountOut, uint256 amountOutAfterFees, uint256 feeBasisPoints)",
```

Updating the .spec's event signature as well

Edit: Also, we can remove either this or the `MOCK_SWAP_EVENT`, right?